python/ml4ir/base/pipeline.py

```diff
@@ -400,6 +400,9 @@ def run(self):
         experiment_tracking_dict.update(train_metrics)
         experiment_tracking_dict.update(test_metrics)
 
+        # Add optimizer and lr schedule
+        experiment_tracking_dict.update(relevance_model.model.optimizer.get_config())
```

Comment:
@mohazahran Does this add both optimizer and learning rate schedule params? It will be useful to do that.

Reply:
Yes it adds both.